addon-test-support/index.js

```diff
@@ -5,8 +5,23 @@ const FAILED_ASSERTION_MESSAGE =
   'One or more objects registered disposables that were not correctly disposed of. Please ensure that objects correctly run their registered disposables by calling `runDisposables` in the `destroy` method of the object.';
 let setupTestDone = false;
 
-export default function setupLifelineValidation(hooks) {
-  let registeredDisposables = new Map();
+/**
+ * Test helper to assert that all async work is disposed of.
+ *
+ * @method setupLifelineValidation
+ * @param { QUnit.hooks } hooks Qunit's hooks object.
+ * @param { Map } options.map Optional map object to use for external reference.
```

Comment:
Should be `options._map` (underscored to make it clear that this is definitely private API)...